internal/gengapic/genrest.go

```diff
@@ -67,25 +76,78 @@ func (g *generator) genRESTMethods(serv *descriptor.ServiceDescriptorProto, serv
 	return nil
 }
 
+func (g *generator) restClientOptions(serv *descriptor.ServiceDescriptorProto, servName string) error {
+	p := g.printf
+
+	var host string
+	if eHost, err := proto.GetExtension(serv.Options, annotations.E_DefaultHost); err == nil {
+		host = *eHost.(*string)
+	} else {
+		fqn := g.descInfo.ParentFile[serv].GetPackage() + "." + serv.GetName()
+		return fmt.Errorf("service %q is missing option google.api.default_host", fqn)
+	}
+
+	p("func default%sRESTClientOptions() []option.ClientOption {", servName)
+	p("  return []option.ClientOption{")
+	p("    internaloption.WithDefaultEndpoint(%q),", host)
```

Comment:
Do we need an import for `internaloption`? I see one in gengrpc.go, but not in genrest.go.
(Also, I seem to be missing this in the `*.want` golden files.)